Sources/GRPC/AsyncAwaitSupport/GRPCAsyncBidirectionalStreamingCall.swift

```diff
@@ -36,8 +36,8 @@ public struct GRPCAsyncBidirectionalStreamingCall<Request: Sendable, Response: S
   }
 
   /// Cancel this RPC if it hasn't already completed.
-  public func cancel() async throws {
-    try await self.call.cancel().get()
+  public func cancel() {
+    self.call.cancel(promise: nil)
```

Comment:
I wonder if, we should make the underlying promise succeed always in a follow up pr?